lib/src/appengine_internal.dart

```diff
@@ -179,7 +180,9 @@ Future<ContextRegistry> _initializeAppEngine() async {
       await _obtainLoggerFactory(context, serviceAccount, zoneId);
   final storageService =
       await _obtainStorageService(context.applicationID, serviceAccount);
-  final memcacheService = await _obtainMemcacheInstance();
+
+  final memcacheService =
+      await _obtainMemcacheInstance(loggerFactory.newBackgroundLogger());
```

Comment:
Assuming accessing the background logger here is "safe"...

Reply:
Yes, I think it should be safe.